test/selenium/src/lib/element/widget_info.py

```diff
@@ -33,12 +36,13 @@ def select_open(self):
     """Select open button in 3BBS dropdown modal."""
     base.Button(self._driver, self.open_locator).click()
 
-  def is_open_exist(self):
-    """Find open button in 3BBS dropdown modal.
-    Return: True if open button is exist,
-            False if open button is not exist.
+  def is_open_enabled(self):
+    """Find open button in 3BBS dropdown modal and check if enabled.
```

Comment:
Re-implement this method by introducing reusable methods in `selenium_utils.py` module
```python
def get_parent(driver, element):
  """Get parent element of curent element using JS."""
  return driver.execute_script("return arguments[0].parentNode;", element)

def is_value_in_parent_attr(dirver, element, attr, value):
  parent = get_parent(dirver, element)
  return is_value_in_attr(parent, attr, value)
```